cde-pentaho5/resource/resources/custom/amd-components/Dashboard/DashboardComponent.js

```diff
@@ -78,9 +78,20 @@ define([
             if(myself.isParameterPublic(otherParam)) {
               var eventName = myParam + ":fireChange";
               var fun = function(evt) {
-                reqDash.fireChange(otherParam, evt.value);
+                if(reqDash.getParameterValue(otherParam) !== evt.value) {
+                  reqDash.fireChange(otherParam, evt.value);
+                }
               };
               myself.dashboard.on(eventName, fun);
+              var subDashboardEventName = otherParam + ":fireChange";
+              var subDashboardfun = function (evt) {
+                if(myself.dashboard.getParameterValue(myParam) !== evt.value) {
+                  if(myself.oneWayMap === false) {
```

Comment:
Was this intended in order to not activate this new behavior in already created dashboards? If not why not just `!myself.oneWayMap` ? Also this should be checked first, as it seems the "lighter" check.